package/firmware/ipq-wifi/Makefile

```diff
@@ -37,7 +37,9 @@ ALLWIFIBOARDS:= \
 	pakedge_wr-1 \
 	qxwlan_e2600ac-c1 \
 	qxwlan_e2600ac-c2 \
-	teltonika_rutx
+	teltonika_rutx \
+        zte_mf289f
+
```

Comment:
Please drop new line 42, and also use tabs instead of spaces in all files. I see Makefile, 02_network and genneric.mk are still align with spaces.

Reply:
fixed, i'll publish in the next commit when we agree on DTS comment about leds